vector/v.net.salesman/main.c

```diff
@@ -500,8 +500,8 @@ int main(int argc, char **argv)
 
             /* tmpcost must always be > 0 */
 
-            G_debug(2, "? %d - %d cost = %f x %f", node1, node2, tmpcost,
-                    cost);
+            G_debug(2, "? %d - %d cost = %f x %f", cities[cycle[j]], cities[cycle[j+1]], // 'node1' and 'node2' is uninitialized when used here
```

Comment:
I'm not quite certain of the correctness of this fix.
@metzm, may I ask you to jump in on this?

Reply:
I have now removed comments and the PR is now ready to be merged. Only an opinion from someone initiated on this particular solution remains...

Reply:
Thanks!